### PR TITLE
Move ffmpeg download url to github repo

### PR DESCRIPTION
**Pull request by misl6**
Moved download url to github repo, in order to speed-up the download. (Current ffmpeg.org download link is freaking slow)

### CODE REVIEW

File: pythonforandroid/recipes/ffmpeg/__init__.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 
 class FFMpegRecipe(Recipe):
-    version = '3.4.5'
-    url = 'http://ffmpeg.org/releases/ffmpeg-{version}.tar.bz2'
+    version = 'n3.4.5'
+    url = 'https://github.com/FFmpeg/FFmpeg/archive/{version}.zip'
```
Comment (AndreMiras) [inline pythonforandroid/recipes/ffmpeg/__init__.py]: shouldn't we have the `n` here `https://github.com/FFmpeg/FFmpeg/archive/n{version}.zip` and then removed from the version?

Comment (misl6) [reply pythonforandroid/recipes/ffmpeg/__init__.py]: Mmm, IMHO We should have the `n` in the version, so, if the user wants to use a specific version / hash from requirements he didn't have to edit also the ffmpeg recipe.

Comment (AndreMiras) [reply pythonforandroid/recipes/ffmpeg/__init__.py]: OK :+1: 
Would you mind adding a comment above the `url` maybe, specifically stating we're using github over ffmpeg.org for performance reasons. That way we don't revert it later if we forget why it's like that

Comment (misl6) [reply pythonforandroid/recipes/ffmpeg/__init__.py]: Sure! Added.

**Pull request merged**